docs/developer_documentation/ref-rest2.md

```diff
@@ -81,6 +81,68 @@ GET http://molgenis.mydomain.example/api/v2/<entity_name>?aggs=x==attr0;y==attr1
 GET http://molgenis.mydomain.example/api/v2/<entity_name>?aggs=x==attr0;y==attr1;distinct=attr2&q=attr4==val
 ```
 
+## Including categorical options
+
+For simple lookup lists, it might be a convenience to include the list in the initial API response.
+You can do this for CATEGORICAL, and CATEGORICAL_MREF attributes using `includeCategories`.
+
+Not including the query option will set it to the default `false`.
+
+Key | Type | Description
+--- | --- | ---
+*includeCategories* | boolean | Includes a list of categorical options in attribute metadata for CATEGORICAL and CATEGORICAL_MREF attributes
+
+## Example request - response
```

Comment:
i suggest a smaller header / or no header, as this is a subsection of the Including categorical options section